src/phew/server.py

```diff
@@ -116,73 +58,21 @@ def __str__(self):
 body: {self.body}"""
 
 
-content_type_map = {
-    "html": "text/html",
-    "jpg": "image/jpeg",
-    "jpeg": "image/jpeg",
-    "svg": "image/svg+xml",
-    "json": "application/json",
-    "png": "image/png",
-    "css": "text/css",
-    "js": "text/javascript",
-    "csv": "text/csv",
-}
-
-
-class FileResponse(Response):
-    def __init__(self, file, status=200, headers={}):
-        self.status = 404
-        self.headers = headers
-        self.file = file
-
-        try:
-            if (os.stat(self.file)[0] & 0x4000) == 0:
-                self.status = 200
-
-                # auto set content type
-                extension = self.file.split(".")[-1].lower()
-                if extension in content_type_map:
-                    headers["Content-Type"] = content_type_map[extension]
-
-            headers["Content-Length"] = os.stat(self.file)[6]
-        except OSError:
-            return False
-
-
 class Route:
-    def __init__(self, path, handler, methods=["GET"]):
+    def __init__(self, path, handler, method="GET"):
         self.path = path
-        self.methods = methods
+        self.method = method
```

Comment:
methods is now a string instead of a list. You can only have one now, but we never had more than one AFAIK. 

we could probably take this further and make methods strictly get or post and save it as a bool. that would save us some more bytes-per-route